### ISSUE TITLE
Page range selection popup in search does not disappear on mobile

### ISSUE THREAD

**Issue by chalkos** [keeps/roda#1399]
Only on mobile. Using chrome with mobile mode also works.

1. navigate to a list with more than 1 page of items
2. select all
3. (the popup will appear) select all in all pages
4. start new process

The popup is still visible and it should not be.

**Comment by chalkos** [comment]
The way I fixed this is not amazingly good.

It is good because it fixes the problem, but it is bad because it only fixes the problem in the specific situation where you are leaving the page that has the list with the selection popup showing. Other cases, like sorting or searching again, change the selection status but the popup will still be visible.

This is a consequence of this component not being mobile friendly, since it is shown/hidden according to mouseover states which are not triggered in mobile browsers (at least not in desktop chrome using mobile mode).

I tried using only focus events to control the popup visibility, and while that works on mobile, it makes the whole thing feel awkward to use on desktop, so I ended up not committing that code.
